demos/fashion-ecommerce/config/category.csv

```diff
@@ -373,7 +378,7 @@ Snickers,Sneakers
 Trainers,Sneakers
 sniker,Sneakers
 snikers,Sneakers
-basketball shoes,Sneakers
+Low tops Low top sneakers basketball shoes,Sneakers
```

Comment:
This line looks a bit odd... As if it should be two lines but they are now collapsed to a single one. Could this be due to some problem in the filter_config.csv? Maybe there's a line break missing or on some line there's a space instead of a tab...

Reply:
The issue was a tab that was missing. Also, I added one more column (alias.9).